### PR TITLE
Created CrawlerBase for creators.com

### PR DESCRIPTION
**Pull request by datagutten**
I created a CrawlerBase for creators.com and updated 'libertymeadows' to use it. I also used it to create a crawler for 'dogsofckennel'.

### CODE REVIEW

File: comics/aggregator/crawler.py [hunk 1]
```diff
@@ -277,3 +278,20 @@ def crawl_helper(self, short_name, pub_date):
...
         return CrawlerImage(url)
+
+
+class CreatorsCrawlerBase(CrawlerBase):
+    """Base comics crawler for all comics posted at creators.com"""
+    headers = {'User-Agent': 'Mozilla/5.0'}
+
+    def crawl_helper(self, feature_id, pub_date):
+        url = 'https://www.creators.com/api/features/get_release_dates?'
+        url += 'feature_id=%s&year=%s' % (feature_id, pub_date.strftime('%Y'))
```
Comment (jodal) [inline comics/aggregator/crawler.py]: I think this is prettier:

``` python
url = (
    'https://www.creators.com/api/features/get_release_dates?'
    'feature_id=%s&year=%s'
) % (feature_id, pub_date.year)
```

**Comment by jodal** [comment]
:+1: with comments

**Comment by datagutten** [comment]
Thanks for showing me a nicer way to do it. I wondered how to avoid the flake8 line length limit.

**Comment by jodal** [comment]
Thanks!

**Pull request merged**